src/plugins/hooks.js

```diff
@@ -6,7 +6,7 @@ const utils = require('../utils');
 
 module.exports = function (Plugins) {
 	Plugins.deprecatedHooks = {
-
+		'filter:user.profileMenu': 'filter:accounts.profile',
```

Comment:
One last thing then it's good to merge... this hook name is too vague, we need a hook name that is indicative that it is for building out buttons. Ping me on slack.

Reply:
`filter:buildLinks.accounts/profile` please